ngi_reports/stockholm/project_summary.py

```diff
@@ -76,94 +80,120 @@ def __init__(self, config, LOG, working_dir, **kwargs):
         self.project_info['ordered_reads'] = self.get_ordered_reads()
         self.project_info['best_practice'] = False if self.proj_details.get('best_practice_bioinformatics','No') == "No" else True
         self.project_info['status'] = "Sequencing done" if self.proj.get('project_summary', {}).get('all_samples_sequenced') else "Sequencing ongoing"
-        self.project_info['library_construction'] = self.proj_details.get('library_construction_method')
-        self.project_info['flowcells_run'] = []
+        self.project_info['library_construction'] = self.get_library_method()
+        self.project_info['accredit'] = self.get_accredit_info(['library_preparation','sequencing','data_processing','data_analysis'])
 
-        # Collect information about the sample preps
+        ## Collect information about the sample preps
         for sample_id, sample in sorted(self.proj.get('samples', {}).iteritems()):
             self.LOG.info('Processing sample {}'.format(sample_id))
-            # Basic fields from Project database
+            ## Basic fields from Project database
             self.samples_info[sample_id] = {'ngi_id': sample_id}
-            self.samples_info[sample_id]['customer_name'] = sample.get('customer_name')
-            self.samples_info[sample_id]['total_reads'] = sample.get('details',{}).get('total_reads_(m)')
+            self.samples_info[sample_id]['customer_name'] = sample.get('customer_name','')
+            self.samples_info[sample_id]['total_reads'] = sample.get('details',{}).get('total_reads_(m)','')
+            if self.project_info['ordered_reads']:
+                self.samples_info[sample_id]['seq_status'] = ['FAIL','PASS'][float(self.samples_info[sample_id]['total_reads']) > \
```

Comment:
come on it is not that big, you should see @mariogiov 's code :stuck_out_tongue: